### ISSUE TITLE
Sharp decrease of primary VOCs between benchmarks in GC12.9 and GC13.0

### ISSUE THREAD

**Issue by axzhang1216** [geoschem/geos-chem#2221]
### Name and Institution (Required)

Name: Aoxing Zhang
Institution: Southern University of Science and Technology

### Confirm you have reviewed the following documentation

- [x ] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [x ] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [ x] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x ] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question

Please provide as much detail as possible. Always include the GEOS-Chem version number and any relevant configuration and log files.

Dear GC support team,

Sorry for bringing up an issue about an old version, but it seems a long story: we are trying to diagnose VOCs in GC14+, then found many primary VOCs (e.g. aromatics) are underestimated, which is ok in the GC12+ simulations. Then we trace back the version history and found from GC12.9 to GC13.0 benchmarks, many primary VOC decreased more than 75% in many regions, for example in BENZ:

![cc18e85bf8038d48a55a15eeab725a4](https://github.com/geoschem/geos-chem/assets/47006350/14015a3d-3385-47d5-a392-21b5b417e065)

We then checked emissions-the emissions of these species are similar between the versions. 
OH concentrations-similar
Rate of the aromatics+OH reaction: not changed

Budgets: the chemical loss in GC13 is significantly more than GC12.9: (e.g. TOLU)
![78bbb6c0b53644acddbccb249058a1d](https://github.com/geoschem/geos-chem/assets/47006350/174990cc-0535-4b27-bc01-b2c5d3df9c8a)

Then we stuck and wonder why it is happening, but we cannot find more clues about the inconsistancy between the versions. Some guesses are (1) the carbon_based unit was removed in GC13.0, (2) some hidden issue changed. 

The species that have similar issues include: EOH, ALK4,BENZ,TOLU,XYLE,C2H6,C3H8,PRPE. 

Hope to see your suggestions!

**Comment by yantosca** [comment]
Thanks for writing @axzhang1216.  This is a science question, which is a bit out of our expertise.  I am tagging the Chemistry Working Group Co-Chairs:  @barronh @luhu0 @jingqiumao

In the meantime I would look at the wiki pages for versions 12 and 13, as the changes each version (and the benchmark plots) should be documented there.  You can get to these links from our [GEOS-Chem versions](https://wiki.seas.harvard.edu/geos-chem/index.php/GEOS-Chem_versions) wiki page.

**Comment by barronh** [comment]
I agree with @yantosca that you should look at the GEOS-Chem versions benchmark notes. Usually, if a species changed by more than 5% there is a note about why it happened. I don't recall this change in particular. Thanks for looking into it.

**Comment by barronh** [comment]
BTW - if it is a carbon issue, that is hopefully just an artifact of how things were processed. In that case, the actual concentration in C should be unchanged.

**Comment by axzhang1216** [comment]
Thanks @yantosca and @barronh for the helpful suggestions!

I checked the version notes (attached below), where I think the carbon unit change may be the closest to this issue, but not sure... 
![image](https://github.com/geoschem/geos-chem/assets/47006350/f286ed62-3421-4275-ae25-ac7e6c9a2ff0)
![image](https://github.com/geoschem/geos-chem/assets/47006350/54a081f7-d337-4d30-91a7-ebadafc979b1)

There were a lot of changes improving the GC structures, but I'm not sure whether can they affect this detailed issue. There were also changes in the emission inventories, but the primary VOC emission didn't show simulaneous change, e.g. for BENZ, the emission increased ~20% in GC13:
![image](https://github.com/geoschem/geos-chem/assets/47006350/c65e910e-fb48-41de-bf87-f5ec247e7dbc)


I noticed that there are lists and explaination for variables that changed more than 10%, as mentioned by @barronh . The note mentioned the unit change in detail, but also mentioned that regardless of the unit conversion, there are species that still have large difference between 13.0 and 12.9 (which seems to be my major concern right now):
![image](https://github.com/geoschem/geos-chem/assets/47006350/cc1f48cd-d1d6-4716-b542-0ae0c85be306)
The note makes sense that the carbon unit update cannot actually explain this. For example BENZ (attached in my last comment), the carbon mass-to-molecule mass is 72:78, which should not cause a ~75% change. So it seems that the team noticed that there maybe other issues related to this primary VOC changes.  

Hope to see any helpful comments on this! I'm feeling that a diagnose may still be needed because many of the primary VOC are similar from benchmark 13.0.0 to the current 14.2 version.

**Comment by yantosca** [comment]
Thanks for the updates @axzhang1216 @barronh.  Also in 13.0.0 was the retirement of the tropchem mechanism and the implementation of the fullchem mechanism.  Furthermore there was a met field change from GEOS_FP to MERRA-2, which may have affected the chemistry as well.

Unfortunately the GCST does not have the resources to diagnose issues in past versions, so we cannot provide much more assistance in this matter.  Thanks for your understanding.

**Comment by barronh** [comment]
@axzhang1216 - My recommendation is to focus on diagnosing the present version.

I think you'll find lots of oddities in the diagnostics during that switch. For example, I just tried to check the TOLU [emissions totals](http://ftp.as.harvard.edu/gcgrid/geos-chem/1yr_benchmarks/13.0.0-beta.1/GCClassic/FullChem/BenchmarkResults/Tables/Emission_totals.txt) and found something odd. The sum of components in the Dev case matches the total (8.255002 + 1.135135 + 0.04854 = 9.438677 ~ 9.438672) but the Ref case doesn't (7.264776 + 0.620118 + 0.044558 = 7.929452 < 9.070378). The difference is larger than the 92.14 / (7 * 12) = 1.0969.

I looked closely at budget files, and didn't see the types of changes that you're suggesting. The file you're using is from a 1-month benchmark for a 13.0.0-beta.0 that didn't make it to the 1-year benchmark. Not only that, but the benchmark year was changing. If you look at the name of the file where you got your budget, it was 2016 for 12.9.0 and 2019 for 13.0.0. When I look at the comparable budget for TOLU from 13.0.0-beta.1 1-year benchmark, the differences are smaller than you noted. I don't know off hand if that benchmark was comparing 2016 for 12.9.0 and 2019 for 13.0.0.

The point is that your time is better spent trying to understand/improve the current version.
